src/Lykke.Logs/LykkeLogToAzureSlackNotificationsManager.cs

```diff
@@ -43,10 +72,7 @@ public void SendNotification(LogEntity entry)
         {
             try
             {
-                if (entry.Level != LykkeLogToAzureStorage.ErrorType
-                    && entry.Level != LykkeLogToAzureStorage.FatalErrorType
-                    && entry.Level != LykkeLogToAzureStorage.WarningType
-                    && entry.Level != LykkeLogToAzureStorage.MonitorType)
+                if (_logLevels.All(l => l != entry.Level))
```

Comment:
Using index (as enum is an integer type) could be faster but maybe harder to understand